src/events/express.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable prefer-rest-params */
+/**
+ * @fileoverview Handlers for Express instrumentation
+ */
+
+const shimmer = require('shimmer');
+const tryRequire = require('../try_require.js');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const expressRunner = require('../runners/express.js');
+
+const express = tryRequire('express');
+
+
+/**
+ * Wraps the Express module request function with tracing
+ * @param {Function} wrappedFunction The Express's request module
+ * @returns {Function} The wrapped function
+ */
+function expressWrapper(wrappedFunction) {
```

Comment:
It's not similar to any wrapper in the wrappers dir, since it's not manual. the name wrapper that written here is because it wraps a function (this name is similar to all other `events/*` files)

Reply:
It's a wrapper, like a wrapper for lambda. It just happened to work with a patch. It's definitely not like other events in this dir.